packages/dialtone-vue2/components/toast/layouts/toast_layout_alternate.vue

```diff
@@ -123,18 +169,22 @@ export default {
       type: Boolean,
       default: false,
     },
-
-    /**
-     * Hides the action from the notice
-     * @values true, false
-     */
-    hideAction: {
-      type: Boolean,
-      default: false,
-    },
   },
 
-  methods: {
+  emits: ['close'],
```

Comment:
it looks like we're not actually emitting 'close' anywhere. Can remove the emits if we're not directly emitting close via $emit()